packages/twenty-server/src/engine/api/graphql/workspace-resolver-builder/interfaces/workspace-resolvers-builder.interface.ts

```diff
@@ -62,7 +62,7 @@ export interface UpdateManyResolverArgs<
   Filter = any,
 > {
   filter: Filter;
-  data: Data;
+  data: Data[];
```

Comment:
data should not be an array here, this is how we use it
```
{
  "filter": {"id": {"in": ["something", "something-else"]}}, 
  "data": {
    "name": "eae"
  }
}
```
where data is the shape of what you want to update against all the records that match with the filters